### PR TITLE
Remove Heretic BLOCKMAP limit

### PR DESCRIPTION
**Pull request by jgreen14**
Port crispy-doom blockmap limit removal code to crispy-heretic

### CODE REVIEW

File: src/heretic/p_local.h [hunk 1]
```diff
@@ -241,8 +241,8 @@ void P_RadiusAttack(mobj_t * spot, mobj_t * source, int damage);
 // ***** P_SETUP *****
 
 extern byte *rejectmatrix;      // for fast sight rejection
-extern short *blockmaplump;     // offsets in blockmap are from here
-extern short *blockmap;
+extern long* blockmaplump;	    // offsets in blockmap are from here
```
Comment (fabiangreffrath) [inline src/heretic/p_local.h]: Please avoid the use of type `long`, it means different things on different platforms. Also, please keep the `type *name` nomenclature for pointers. In general, you should make sure the code you are going to introduce in Crispy Heretic today looks like the pendant code in Crispy Doom *today*, not at the time of the initial commit.

Comment (jgreen14) [reply src/heretic/p_local.h]: Makes sense. I of course was just using the original commit as the guide for the patch, but I can bring it into line with the current state of Crispy Doom.

**Comment by jgreen14** [comment]
@fabiangreffrath Just letting you know that I updated the blockmap limit removal code to remove longs and to make it match the blockmap limit removal code in crispy-doom.

**Comment by fabiangreffrath** [comment]
I have seen that, thanks!

**Comment by fabiangreffrath** [comment]
Cool, thanks!

**Pull request merged**